### PR TITLE
[DOC][FIX]Bad directory reference

### PR DESCRIPTION
**Pull request by flotho**

### CODE REVIEW

**Comment by flotho** [comment]
ping @StefanRijnhart

File: docsource/development.rst [hunk 1]
```diff
@@ -52,7 +52,7 @@ Learn from existing migration scrips
 
 
 Since version 14, the migration scripts are located in:
-``OpenUpgrade/openupgrade-framework/openupgrade_scripts/scripts/<module-name>``
+``OpenUpgrade/openupgrade_scripts/scripts/<module-name>``
```
Comment (StefanRijnhart) [inline docsource/development.rst]: It probably makes sense to drop the `OpenUpgrade` part here, analogous to the mention above.

**Comment by flotho** [comment]
better @StefanRijnhart  ?

**Review by StefanRijnhart — APPROVED** [review]
Thanks!

File: docsource/development.rst [hunk 2]
```diff
@@ -16,7 +16,7 @@ steps are not necessary):
 - end-migration.py
 
 Since version 14, these files can be found here:
-``openupgrade-framework/openupgrade_scripts/scripts/<module-name>``
+``openupgrade_scripts/scripts/<module-name>``
```
Comment (pedrobaeza) [inline docsource/development.rst:20]: Now that you are here, it seems the initial intention was to put this into a new paragraph, but for proper RST syntax for that, you need to leave an extra empty line (the same for both changes).

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**